integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
```

Comment:
The other change I made to this package file was making `react-scripts` an optional dependency, as it's not supported in Node 4. We aren't running the integration tests in Node 4, so it doesn't actually matter if it gets installed.

We've run into this several times with packages. I'm fine continuing to whitelist packages like this, but do you guys know whether we actually need to support Node 4?